src/rdkb/include/cpeabs_rdkb.h

```diff
@@ -0,0 +1,48 @@
+/* SPDX-FileCopyrightText: 2021 Comcast Cable Communications Management, LLC */
```

Comment:
The cpeabs.h file should really hide this detail from being exported.  These functions should be normalized and added there vs. having a `cpeabs_rdkb.h` and a `cpeabs_pod.h` and other files that are used by other libraries.  This can live in the src directory if it is only needed internally.